index.html

```diff
@@ -3,7 +3,7 @@
 <head>
 	<meta charset="UTF-8">
 	<title>NPM Build System w/ Less</title>
-	<link rel="stylesheet/less" href="css/index.less">
+	<link rel="stylesheet" href="npm-build-system.1.0.0.css">
```

Comment:
You were trying to link to a less file.  Browsers don't understand less, they have to be pre-processed into css.  This means that you have to link to the generated css file.  Luckily you know what that is going to be based on the `build:css` task you have: `build/$npm_package_name.$npm_package_version.css` Where `$npm_package_name` is `npm-build-system`, so you know that the file name is going to be `npm-build-system.1.0.0.css`.